### PR TITLE
makeSearchBar

### PR DESCRIPTION
**Pull request by serhii-vorona**
- [DEMO LINK](https://serhii-vorona.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://serhii-vorona.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

**Comment by serhii-vorona** [comment]
> * [DEMO LINK](https://serhii-vorona.github.io/layout_search-bar-airbnb/)
> * [TEST REPORT LINK](https://serhii-vorona.github.io/layout_search-bar-airbnb/report/html_report/)

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,48 @@
...
+        <input
+            id="bigSearch"
+            type="search"
+            class="search bigSearch fokyc"
+            data-qa="keypress"
+            placeholder="Try &quot;Los Angeles&quot;"
+          >
+      </form>
+    </div>
+    <div class="container containerSmall">
```
Comment (Olexandr6) [inline src/index.html:36]: Add a blank line between these elements.

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,48 @@
...
+        data-qa="big"
+      >
+        <label for="bigSearch">
+          <img
+            src="./images/Search.svg"
+            class="bigIcon"
+            alt="icon"
+          >
+        </label>
+        <input
```
Comment (Olexandr6) [inline src/index.html:27]: Add a blank line between these elements.

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,48 @@
...
+        data-qa="small"
+      >
+        <label for="smallSearch">
+          <img
+            src="./images/Search.svg"
+            class="smallIcon"
+            alt="icon"
+          >
+        </label>
+        <input
```
Comment (Olexandr6) [inline src/index.html:49]: Add a blank line between these elements.

File: src/index.html [hunk 4]
```diff
@@ -11,6 +11,48 @@
...
+        id="smallSearch"
+        type="search"
+        class="search smallSearch fokyc"
+        placeholder="Try &quot;Los Angeles&quot;"
```
Comment (Olexandr6) [inline src/index.html:53]: Add indents. Check 4th point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+  border-radius: 4px;
+}
+
+.containerBig {
+  margin-top: 20px;
+}
+
+.containerSmall {
+  margin-top: 20px;
+}
```
Comment (Olexandr6) [inline src/style.css:32]: Don't duplicate styles. Check 2nd point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+.smallSearch {
+  width: 100%;
+  height: 42px;
+  font-size: 14px;
+  padding-left: 33px;
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+}
```
Comment (Olexandr6) [inline src/style.css:50]: Check 2nd point from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+  width: 11px;
+  position: absolute;
+  padding-top: 15px;
+  padding-left: 13px;
+  padding-bottom: 16px;
+}
+
+/* .smallSearch::placeholder {
+  font-weight: 300;
+} */
```
Comment (Olexandr6) [inline src/style.css:68]: Don't leave redundant comments. Delete this

**Review by Olexandr6 — COMMENTED** [review]
Don't close your pull request. Looks like you don't even see the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md).

Check comments👇

**Review by serhii-vorona — COMMENTED** [review]
- [DEMO LINK](https://serhii-vorona.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://serhii-vorona.github.io/layout_search-bar-airbnb/report/html_report/)

**Comment by serhii-vorona** [comment]
> * [DEMO LINK](https://serhii-vorona.github.io/layout_search-bar-airbnb/)
> * [TEST REPORT LINK](https://serhii-vorona.github.io/layout_search-bar-airbnb/report/html_report/)

**Comment by serhii-vorona** [comment]
> * [DEMO LINK](https://serhii-vorona.github.io/layout_search-bar-airbnb/)
> * [TEST REPORT LINK](https://serhii-vorona.github.io/layout_search-bar-airbnb/report/html_report/)

File: src/index.html [hunk 8]
```diff
@@ -10,7 +10,53 @@
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
```
Comment (mykola-koval) [inline src/index.html:15]: do you really need this div? you can style the form directly

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 400;
+}
+
+.search {
+  background: rgb(255, 255, 255);
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
```
Comment (mykola-koval) [inline src/style.css:21]: Always add `box-sizing: border-box;` on all elements `* {...}`

**Review by mykola-koval — APPROVED** [review]

**Pull request closed without merging**